TLM/TLM/Custom/PathFinding/CustomPathFind.cs

```diff
@@ -298,11 +298,11 @@ private struct BufferItem {
                 (queueItem_.vehicleType & ExtVehicleType.RoadVehicle) != ExtVehicleType.None;
 
             isLaneArrowObeyingEntity_ =
-                (!Options.relaxedBusses || queueItem_.vehicleType != ExtVehicleType.Bus) &&
+                (!SavedGameOptions.Instance.relaxedBusses || queueItem_.vehicleType != ExtVehicleType.Bus) &&
                 (vehicleTypes_ & LaneArrowManager.VEHICLE_TYPES) != VehicleInfo.VehicleType.None &&
                 (queueItem_.vehicleType & LaneArrowManager.EXT_VEHICLE_TYPES) != ExtVehicleType.None;
 #if DEBUG
-            isLaneArrowObeyingEntity_ &= !Options.allRelaxed;
+            isLaneArrowObeyingEntity_ &= !SavedGameOptions.Instance.allRelaxed;
```

Comment:
Hmm, maybe cache that static in `PathFindImplementation()` just like `globalConf_`?

Reply:
`Instance` is inlined though. is that not enough?

Reply:
I seriously doubt optimizations like this would have any noticeable influence on performance.

Reply:
It's not about optimization, but `NullReferenceException` prevention...

Reply:
@krzychu124 
I may have not compiled the options because now I am getting this 
```
Info 53.1619332: SavedGameOptions.Instance is NOT inlined
```
also game does not load.

Reply:
yeah, I already posted stacktrace with log if you need to look inside. I suppose it passed `null` as argument but I'm not sure why - I see Ensure call before 🤷‍♂️

Reply:
I tested wrong version of TMPE last time 🤦‍♂️ 
The issue was that I was forgot to replace null with Instance